script/urlChecker.ts

```diff
@@ -74,6 +91,23 @@ async function checkPages(bookSlug: string, pages: string[], redirects: Redirect
   return anyFailures || notFound.length > 0;
 }
 
+const loadRedirectedPagesForBookId = async(bookID: string) => {
+  const fileName = path.resolve(__dirname, `../data/redirects/${bookID}.json`);
+  if (!fs.existsSync(fileName)) { return; }
```

Comment:
you can return an empty array here and you will not need and `if` statement in `visitPage`